_opcodes/op232-bgee.html

```diff
@@ -13,37 +13,57 @@
 ---
 Casts the spell specified by the resource field on the target specified by the 'Target' field, when the condition from the 'Condition' field is true. The conditions are akin to various triggers.
 <br/>Known values for 'Target' are:
-<br/>0   Effect Target
+<br/>0   Myself
 <br/>1   LastHitter
 <br/>2   NearestEnemyOf
-<br/>3   Nearest
+<br/>3   NearestNotSelf
```

Comment:
Nearest is much clearer. There's 0 for oneself anyway (at least typically, "effect target" is technically more correct).

Reply:
OK, reverted it back to just "Nearest".